autoarray/structures/grids/two_d/grid_2d_pixelization.py

```diff
@@ -306,25 +306,9 @@ def voronoi(self) -> scipy.spatial.Voronoi:
         except (ValueError, OverflowError, scipy.spatial.qhull.QhullError) as e:
             raise exc.PixelizationException() from e
 
-    @cached_property
-    def pixel_neighbors(self) -> PixelNeighbors:
-        """
-        A class packing the ndarrays describing the neighbors of every pixel in the Voronoi pixelization (see
-        `PixelNeighbors` for a complete description of the neighboring scheme).
-
-        The neighbors of a Voronoi pixelization are using the `ridge_points` attribute of the scipy `Voronoi` object,
-        as described in the method `pixelization_util.voronoi_neighbors_from`.
-        """
-        neighbors, sizes = pixelization_util.voronoi_neighbors_from(
-            pixels=self.pixels, ridge_points=np.asarray(self.voronoi.ridge_points)
-        )
-
-        return PixelNeighbors(arr=neighbors.astype("int"), sizes=sizes.astype("int"))
-
     @cached_property
     def split_cross(self):

```

Comment:
Would it be correct to state that the reason we draw this `split_cross` is in order to estimate the gradient (or derivate?) of the Voronoi pixels after they are asigned numerical values?

I am considering renaming `split_cross` to something like `gradient` as it feels more description.